packages/sdks/src/blocks/util.ts

```diff
@@ -1,6 +1,5 @@
-export function markSerializable<FN extends (...args: any[]) => any>(
```

Comment:
cc @mhevery I think you had some special logic here but stringifying the functions does the trick instead